midudev/midu.dev
# feat: copy button for codeblocks (just in articles) added

## What
Copy Button For Codeblocks

## Why
Select and copy is a painful task for devs ;)

## Testing?
Every codeblock contains in its bottom-right part a button that allows the user copy the codeblock content.

- Go to an article
- Check a codeblock bottom-ritght button
- Click on copy
- Then check copied content pasting it in whatever textbox/document


## Screenshots (optional)
<img width="886" alt="Screenshot 2023-05-20 at 10 53 28 AM" src="https://github.com/midudev/midu.dev/assets/34603733/9415d067-1a90-4d72-81d1-e1fcd882ee69">
